test/export_test.rb

```diff
@@ -58,50 +58,6 @@ def tmp_dir
     assert data["de"]
   end
 
-  test "writes dot-separated symbols to yaml in a way that can be loaded back" do
-    data = { "format" => { "narrow" => :"calendars.gregorian.months.stand-alone.narrow" } }
-    yaml = Cldr::Export::Yaml.new.yaml(data.deep_stringify_keys)
-
-    assert_equal data, YAML.load(yaml)
-  end
```

Comment:
These tests were once useful, but after https://github.com/ruby-i18n/ruby-cldr/pull/43 they became useless, and started testing `Hash#to_yaml`